src/desktop/components/artwork_filter_2/views/header_sorts_view.coffee

```diff
@@ -9,13 +9,19 @@ module.exports = class ArtworkFiltersSortsView extends BorderedPulldown
     # Render the template once when the params are first set.
     # Afterwards, the superclass handles changes in the selection.
     @listenToOnce @params, 'firstSet', @render
-    super
 
-  sorts:
+    @sorts = {
     '-partner_updated_at': 'Recently Updated'
     '-published_at': 'Recently Added'
     '-year': 'Artwork Year (desc.)'
     'year': 'Artwork Year (asc.)'
+    }
+
+    # allows decayed_merch to be set via url.
+    if @params && @params.get('sort') && @params.get('sort') == '-decayed_merch'
```

Comment:
I think you can use the coffeescript `?` to make it nicer (and one line):

`@sorts['-decayed_merch'] = 'Default' if @params?.get('sort') is '-decayed_merch'`

Also I didn't immediately 'get' the approach here, so just want to make sure I understand:
- adding `'-decayed_merch': 'Default'` to the sorts map will make them visible in the drop-down on the front-end?
- so instead, only if explicitly set in the URL (which flows down to `@params`), do you want to include it in the map?

tricksy!

Also, I'm not sure if `@params` here is ever `null` or `undefined`, shouldn't it always be an instance of the 
 params model? (from https://github.com/artsy/force/blob/7f9cfaad1d6aa3dc27f6e23795566947b7010b31/src/desktop/components/artwork_filter_2/view.coffee#L40)

So you may not need the `?` check and can just do `if @params.get('sort') is '-decayed_merch'`